src/components/dashboard/FaceVerification/screens/IntroScreen.js

```diff
@@ -79,8 +83,8 @@ const IntroScreen = ({ styles, screenProps }) => {
   const [, requestCameraPermissions] = usePermissions(Permissions.Camera, {
     requestOnMounted: false,
     onAllowed: openFaceVerification,
-    onPrompt: () => fireEvent(FV_CAMERAPERMISSION),
-    onDenied: () => fireEvent(FV_CANTACCESSCAMERA),
+    onPrompt: () => fireEvent(`${FV_CAMERAPERMISSION}_AB_${AB}`),
```

Comment:
we dont AB test these, plus we dont modify event names, we the AB as params

Reply:
done